docs/footer.html

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+    <!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can check out other license options via our website: themes.3rdwavemedia.com */-->
+    <p>© 2017 <a href="#">Privacy Policy</a> | <a href="#">Terms of Services</a><a href=""> <i class="fa fa-github"
+                                                                                               style="float: right;"
+                                                                                               aria-hidden="true">Github</i></a>
+    </p>
+</div>
```

Comment:
On Github, there was a link to spark-nlp. It was added by me on each page. Kindly add it back here